api/HTMLCanvasElement.json

```diff
@@ -147,6 +147,55 @@
             "deprecated": false
           }
         },
+        "alpha_attribute": {
```

Comment:
This should actually be "options_alpha_parameter" since it represents the "alpha" dictionary member of the second argument to `getContext`.

Unfortunately there are no spec links for these since it's not a Web IDL dictionary but is rather treated as a JavaScript object.